src/test/scala/special/sigma/SigmaDslTest.scala

```diff
@@ -111,6 +111,49 @@ class SigmaDslTest extends PropSpec with PropertyChecks with Matchers with Sigma
     }
   }
 
+  property("byteArrayToLong equivalence") {
+    val eq = checkEq(func[Coll[Byte],Long]("{ (x: Coll[Byte]) => byteArrayToLong(x) }")){ x =>
+      Longs.fromByteArray(x.toArray)
```

Comment:
The Scala lambda should be exactly the same as Script lambda, because it should be SigmaDsl expression, which is the same as SigmaScript expression.
Here we actually test equivalence of SigmaDsl and SigmaScript.

Reply:
Whoa! How can I miss this? Fixed (mostly) in [commit](
https://github.com/ScorexFoundation/sigmastate-interpreter/pull/400/commits/237de80470a2c35de96e02563a486a8c01fb9f1f) 

Now I clearly get why we need prefix methods. :)